resources/assets/components/utilities/Chrome.js

```diff
@@ -14,4 +14,30 @@ const Chrome = ({ title, subtitle, children }) => (
   </div>
 );
 
+const Chrome = ({ title, subtitle, children, error, loading }) => {
```

Comment:
I've never understood why stuff is called Chrome and why there is a browser called Chrome too and it's ungoogleable because I just get results about the browser! Can you explain this to me?

Reply:
Oh haha, yeah that's fair! 😆 It's a [user interface term](https://stackoverflow.com/a/5072092) (and I think at least [partially the namesake for the browser](https://www.quora.com/Why-is-Google-Chrome-browser-named-as-Chrome), but it is kinda confusing now! We could call this `Shell` or `Layout` perhaps to be clearer.

Reply:
OOOOO finally, the mystery is solved. Thanks! I dig `Shell` because it seems more descriptive to me, but a potential change for another day!

Reply:
I renamed to `Shell` in e44e5b53! 🐚